docs/tests/manual-scenarios.md

```diff
@@ -0,0 +1,93 @@
+| TC1.5 Wylogowanie             |                                    | 1. LOGIN: "doesnot"<br>2. HASŁO: "matter"<br>3.ADRES ŚRODOWISKA: "http://localhost:3000/"| 1. Wprowadź login<br>2. Kliknij "Zaloguj"<br>3. Kliknij "Wyloguj"<br>4. Zmień url strony na adres środowiska (bez podkatalogu)   | 1. Przekierowanie na stronę loginu<br>2. Przekierowanie na stronę loginu również po zmianie adresu |
+
+<div style="page-break-after: always;"></div>
+
+**Nazwa scenariusza**: Tworzenie wydarzenia
+
+**ID**: S2
+
+**Opis**: Weryfikacja procesu tworzenia nowego wydarzenia przez organizatora
+
+**Czynności przygotowawcze**: -
+
+**Czynności końcowe**: Usunięcie wydarzenia
+
+**Przypadki testowe scenariusza**:
+
+
+| ID i Nazwa Przypadku Testowego           | Warunki wstępne                                                                                 | Dane testowe                                                                                                           | Kroki reprodukcji                                                                                                                                                                                                                                                                                                                               | Oczekiwany rezultat                                                                     |
+|------------------------------------------|-------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| TC2.1 Utworzenie poprawnego wydarzenia   | 1. Istnieje konto organizatora<br>2. Organizator jest zalogowany<br>3. Istnieje dowolna lokacja | 1. NAZWA: "event1"<br>2. OPIS: "description"<br>3. NAZWA BILETU: "standard"<br>4. CENA: 100<br>5. LICZBA BILETÓW: 1000 | 1. Kliknij przycisk dodawania wydarzenia<br>2. Wprowadź nazwę i opis<br>3. Wprowadź dowolną datę początkową i datę końcową o 1 dzień późniejszą<br>4. Wprowadź dowolne godziny początkowe i końcowe <br>5. Wybierz lokację z listy<br>6. Wybierz dowolny obraz<br>7. Zmień nazwę biletu<br>8. Ustaw cenę i ilość<br>9. Kliknij przycisk "Dodaj" | 1. Utworzenie wydarzenia<br>2. Wyświetlenie komunikatu o pomyślnym utworzeniu           |
```

Comment:
Istnieje dowolna lokacja --> lokalizacja https://pl.wikipedia.org/wiki/Lokacja_(historia)

Reply:
nit: zaczynając kolejne punkty z wielkiej litery prosi się o wstawianie kropki na koniec

Reply:
nie wiem jak bardzo szczegółowo to trzeba opisywać, ale może dodać w pkt. 2. że data musi być nadchodząca a nie w przeszłości?

Reply:
>  lokacja --> lokalizacja

Już zmieniłem. Zmyliło mnie słowo użyte w formularzu

> nit: zaczynając kolejne punkty z wielkiej litery prosi się o wstawianie kropki na koniec

Pozwolę sobie to odpuścić

> może dodać w pkt. 2. że data musi być nadchodząca

Słuszna uwaga. Dodałem też TC na daty w przeszłości